package.json

```diff
@@ -54,31 +54,73 @@
     "@types/commander": "^2.12.2",
     "@types/configstore": "^4.0.0",
     "async-mutex": "^0.2.4",
+    "axios": "^0.19.0",
+    "body-parser": "^1.19.0",
+    "bunyan": "^1.8.14",
+    "chai": "^4.2.0",
+    "chai-as-promised": "^7.1.1",
     "chalk": "^4.0.0",
     "commander": "^5.1.0",
     "configstore": "^5.0.1",
-    "connect": "^3.7.0",
+    "connect-ensure-login": "^0.1.1",
+    "cookie-parser": "^1.4.5",
+    "cors": "^2.8.5",
+    "ejs": "^3.1.5",
     "encryptedfs": "^1.0.2",
+    "express": "^4.17.1",
+    "express-oauth-server": "^2.0.0",
+    "express-openapi-validator": "^4.0.4",
+    "express-session": "^1.17.1",
+    "form-data": "^2.3.3",
     "google-protobuf": "^4.0.0-rc.2",
     "isomorphic-git": "^1.7.4",
+    "js-yaml": "^3.3.0",
+    "jsonwebtoken": "^8.5.1",
+    "jstoxml": "^1.5.0",
     "kbpgp": "^2.0.82",
     "node-forge": "^0.9.1",
+    "node-yaml-config": "0.0.6",
+    "oauth2orize": "^1.11.0",
+    "ono": "^5.0.1",
+    "openapi-sampler": "^1.0.0-beta.15",
     "pako": "^1.0.11",
+    "passport": "^0.4.1",
+    "passport-http": "^0.3.0",
+    "passport-http-bearer": "^1.0.1",
+    "passport-local": "^1.0.0",
+    "passport-oauth2": "^1.5.0",
+    "passport-oauth2-client-password": "^0.1.2",
```

Comment:
Are all these passport libraries needed? Especially for the limited usecase of OAuth that we are using?

Reply:
since were now only using bearer tokens and client password flows (I left this one in since it is just another way to get a token and is nothing like the other ways of doing oauth like facebook or github logins.) I will remove the others